xml/wgl.xml

```diff
@@ -736,8 +734,7 @@ Registry at https://github.com/KhronosGroup/OpenGL-Registry
             <param><ptype>HANDLE</ptype> <name>hDevice</name></param>
             <param len="1">void *<name>dxObject</name></param>
             <param><ptype>GLuint</ptype> <name>name</name></param>
-            <!-- FIXME: reference GL enums? -->
-            <param ><ptype>GLenum</ptype> <name>type</name></param>
+            <param group="gl::WGLObjectTypeDX"><ptype>GLenum</ptype> <name>type</name></param>
```

Comment:
`wgl::ObjectTypeDX`?

Reply:
Ah yeah missed this one, I originally created the enum group in `gl::` but changed it later. Will fix